cirq-ionq/README.rst

```diff
@@ -0,0 +1,27 @@
+.. image:: https://upload.wikimedia.org/wikipedia/en/thumb/d/d4/IonQ_corp_logo.svg/2560px-IonQ_corp_logo.svg.png
+  :target: https://github.com/quantumlib/cirq/
+  :alt: cirq-ionq
+  :width: 500px
+
+`Cirq <https://quantumai.google/cirq>`__ is a Python library for writing, manipulating, and optimizing quantum
+circuits and running them against quantum computers and simulators.
+
+This module is **cirq-ionq**, which provides everything you'll need to run Cirq quantum algorithms on IonQ quantum computers.
+
+Documentation
+-------------
+
+To get started with IonQ quantum computing services, checkout the following guide and tutorial:
+
+- `Access and authentication <https://quantumai.google/cirq/ionq/access>`__
+- `Getting started guide <https://quantumai.google/cirq/tutorials/ionq/getting_started>`__
+
+Installation
+------------
+
+To install the stable version of only **cirq-ionq**, use `pip install cirq-ionq`.
+To install the pre-release version of only **cirq-ionq**, use `pip install cirq-ionq --pre`.
+
+Note, that this will install both cirq-ionq and cirq-core as well.
```

Comment:
Super nit:

> Note, that this will install both cirq-ionq and cirq-core as well. To get all the optional modules installed as well

To many `well`s following each other imo. Get rid of the first one.

Reply:
Well spotted :)